client/my-sites/theme/hooks/use-bundle-settings.tsx

```diff
@@ -54,6 +63,28 @@ export function useBundleSettings( themeSoftware?: string ): BundleSettingsHookR
 					),
 				};
 
+			case 'sensei':
+				return {
+					name: 'Sensei LMS',
+					softwareName: 'Sensei LMS',
+					iconComponent: SenseiIcon,
+					color: '#43AF99',
+					designPickerBadgeTooltip: translate(
+						'This theme comes bundled with Sensei. Create and sell courses that your students will love.'
```

Comment:
Almost all the texts, I used the same from WooCommerce, and just changed the name to "Sensei". The only one that is a little different is this one, which I copied the text we have on senseilms.com home.